instrumentation/gwt-2.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/gwt/GwtSingletons.java

```diff
@@ -0,0 +1,29 @@
+package io.opentelemetry.javaagent.instrumentation.gwt;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.context.ContextKey;
+import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter;
+import io.opentelemetry.instrumentation.api.instrumenter.rpc.RpcAttributesExtractor;
+import io.opentelemetry.instrumentation.api.instrumenter.rpc.RpcSpanNameExtractor;
+import java.lang.reflect.Method;
+
+public final class GwtSingletons {
```

Comment:
I think I've used every pattern possible for the name of this sort of class. I wonder if we should just converge on `*Singletons` so it can fit instrumenters, but also any other singletons that come our way.